base/system/services/services.h

```diff
@@ -197,10 +200,11 @@ DWORD ScmCreateNewServiceRecord(LPCWSTR lpServiceName,
 VOID ScmDeleteServiceRecord(PSERVICE lpService);
 DWORD ScmMarkServiceForDelete(PSERVICE pService);
 
-DWORD ScmControlService(HANDLE hControlPipe,
-                        PWSTR pServiceName,
-                        SERVICE_STATUS_HANDLE hServiceStatus,
-                        DWORD dwControl);
+DWORD
+ScmControlService(PSERVICE lpService,
+                  PWSTR pServiceName,
+                  SERVICE_STATUS_HANDLE hServiceStatus,
+                  DWORD dwControl);
```

Comment:
since the formatting here was changed already, let's do it right:
```suggestion
    PWSTR pServiceName,
    SERVICE_STATUS_HANDLE hServiceStatus,
    DWORD dwControl);
```